### DISCUSSION TITLE
command-tab app switcher w/ yabai

### DISCUSSION THREAD

**Discussion by lilruffian** [asmvik/yabai#1029] [Q&A] [answered]
Hey all, 

Just wondering if anyone knows of a way to always use yabai to switch apps/spaces when using macOS build in cmd-tab app switcher? My muscle memory for cmd-tab runs deep, and it's proving to be a difficult habit to break. Is it possible to tell macOS to always switch spaces/apps with yabai instead of mission control when activating via cmd-tab app switcher? The slight delay in mission control changing spaces is so annoying, ha! 
I have SIP disabled.

**Comment by asmvik** [comment, marked as answer]
This delay will be removed when the next version of yabai is released. See #1235 for history.
